src/main/java/org/apache/ibatis/io/DefaultVFS.java

```diff
@@ -189,13 +189,13 @@ public boolean isValid() {
     for (JarEntry entry; (entry = jar.getNextJarEntry()) != null;) {
       if (!entry.isDirectory()) {
         // Add leading slash if it's missing
-        String name = entry.getName();
-        if (!name.startsWith("/")) {
-          name = "/" + name;
+        StringBuilder name = new StringBuilder(entry.getName());
+        if (!name.substring(0).startsWith("/")) {
```

Comment:
As `substring()` creates a new `String`, `name.charAt(0) != '/'` would be more efficient.